tests/unit/matrix/test_assembler.py

```diff
@@ -1574,6 +1579,34 @@ def it_provides_the_order(self, request):
     def it_provides_the_element_values_to_help(self, request, _marginal_prop_):
         assert _SortRowsByMarginalHelper(None, None)._element_values == "a"
 
+    @pytest.mark.parametrize(
+        "marginal, marginal_prop_name",
+        (
+            (MARGINAL.SCALE_MEAN, "rows_scale_mean"),
+            (MARGINAL.SCALE_MEAN_STDDEV, "rows_scale_mean_stddev"),
+            (MARGINAL.SCALE_MEDIAN, "rows_scale_median"),
+        ),
+    )
+    def it_provides_the_marginal_to_help(self, request, marginal, marginal_prop_name):
+        second_order_measures_ = instance_mock(request, SecondOrderMeasures)
+        setattr(second_order_measures_, marginal_prop_name, "foo")
+        _order_spec_ = instance_mock(request, _OrderSpec, marginal=marginal)
+        dimensions_ = [instance_mock(request, Dimension, order_spec=_order_spec_)]
+        order_helper = _SortRowsByMarginalHelper(dimensions_, second_order_measures_)
+
+        assert order_helper._marginal == "foo"
+
+    def but_it_raises_on_unknown_marginal(self, request):
+        second_order_measures_ = instance_mock(request, SecondOrderMeasures)
+        _order_spec_ = instance_mock(request, _OrderSpec, marginal="bar")
+        dimensions_ = [instance_mock(request, Dimension, order_spec=_order_spec_)]
```

Comment:
This is an isolation gap. You're exercising `_RowOrderHelper._order_spec` unnecessarily. What you want is `_order_spec_prop_` and `_order_spec_prop_.return_value = order_spec_`. Dimensions don't come into it for this unit.